security-policies/RULES.md

```diff
@@ -390,9 +390,9 @@
 
 ## AZURE CIS Benchmark
 
-### 44/151 implemented rules (29%)
+### 45/151 implemented rules (30%)
 
-#### Automated rules: 44/77 (57%)
+#### Automated rules: 45/77 (58%)
```

Comment:
I thing this needs to be generated again

Reply:
I run both `dev/update_rule_status.py` and `dev/generate_rule_metadata.py` with no result (they also run through pre-commit). 

Am I missing something in the numbers?

Reply:
Ah, I'm missing it. I thought it would be higher. But this is indeed the right one.